site/assets/bulma.css

```diff
@@ -2,8 +2,7 @@
 
 .cse .gsc-control-cse,
 .gsc-control-cse {
-  padding-top: 0 !important;
-  padding-bottom: 0 !important;
+  padding: 0 !important;
```

Comment:
Can you override this in the `ocrd.css` stylesheet rather than changing the bulma CSS directly, in case we ever want to update the framework?